src/components/cylc/Tree.vue

```diff
@@ -112,11 +119,6 @@ export default {
           this.activeCache.add(treeItem)
         }
       }
-    },
-    clearCaches () {
-      this.treeItemCache.clear()
-      this.activeCache.clear()
-      this.expandedCache.clear()
```

Comment:
This was removing all entries, without re-creating them (I thought they were being re-created). Now we are selective, and I also learned how/where to look for the size of the caches (which helps prevent performance issues).